### PR TITLE
 fix for dubbo source metric destination label unknown

### PR DESCRIPTION
**Pull request by huanghuangzym**
fix for  source  metric destination label is unknown  https://github.com/aeraki-mesh/aeraki/issues/196

### CODE REVIEW

File: src/application_protocols/dubbo/dubbo_codec.cc [hunk 1]
```diff
@@ -77,6 +78,57 @@ void DubboCodec::encode(const MetaProtocolProxy::Metadata& metadata,
...
+      }
+      has_mutation = true;
+    }
+
+    ENVOY_LOG(debug, "dubbo: encodeResponse codec attachment is {}",
+              result->attachment_->attachment().toDebugString());
+  }
+
+  if (has_mutation) {
+    //服务端的response包含了mutation header
```
Comment (zhaohuabing) [inline src/application_protocols/dubbo/dubbo_codec.cc:119]: Please use English to comment

Comment (huanghuangzym) [reply src/application_protocols/dubbo/dubbo_codec.cc:119]: ok,i will fix all chinese comment

**Comment by huanghuangzym** [comment]
image

**Review by zhaohuabing — APPROVED** [review]
LGTM, thanks for the great feature!

**Pull request merged**